pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -26,6 +25,20 @@ def _update_time_to_status_change(self):
         new_time = random.randint(1, 10)
         return new_time
 
+    def _set_latent_time(self):
+        """Calculates and returns latency period as calculated in
+        covid-sim to be given as time until next infection status
```

Comment:
```suggestion
        CovidSim, given as the time until next infection status
```

I found this hard to read but this might be clearer?